zerver/openapi/zulip.yaml

```diff
@@ -4281,6 +4281,21 @@ paths:
                                             **Changes**: New in Zulip 9.0 (feature level 266). Previously
                                             `realm_create_private_stream_policy` field used to control the
                                             permission to create private channels.
+                                    can_create_web_public_channel_group:
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining
+                                            the set of users who have permission to create web-public
+                                            channels in this organization.
+
+                                            This setting can only be set to `"role:moderators"`,
+                                            `"role:administrators"`, `"role:owners"` and
+                                            `"role:nobody"` system groups.
+
+                                            **Changes**: New in Zulip 10.0 (feature level 280). Previously
+                                            `realm_create_web_public_stream_policy` field used to control
+                                            the permission to create web-public channels.
```

Comment:
These changes entries should explain the mapping; otherwise it's a client author doesn't have the information required to translate.

Reply:
Oh, I guess we still have the legacy field values, so maybe that's not required.